registries/models.py

```diff
@@ -409,7 +409,7 @@ class RegistriesApplicationStatus(AuditModel):
     application = models.ForeignKey(
         RegistriesApplication,
         db_column='application_guid',
-        on_delete=models.PROTECT,
+        on_delete=models.CASCADE,
```

Comment:
@garywong-bc this cascade opens up a can of worms. We need to talk about marking records as deleted instead of deleting them, then the protect vs. cascade issue goes away.